docs/modules/eprescription/prescribing/rx-renewal.md

```diff
@@ -7,17 +7,18 @@ It is required that RxRenewal is preceded by NewRx.
 
 ### Receiving RxRenewalRequest
 
-Once the pharmacy sends an RxRenewal, ePrescription module receives Surescripts message at `/eprescriptions/rx` endpoint and saves it to Aidbox.
-The message is converted to a **MedicationRequest** and the creation is registered as a **Provenance** resource.
-There are several ways to track the newly created **MedicationRequest**s:
-- [Either of Aidbox Subscriptions mechanisms](../../topic-based-subscriptions/README.md).
-- Manual/automated tracking based on **MedicationRequest** modification date and/or status.
-- **Provenance** tracking.
+Once the pharmacy sends an RxRenewalRequest, ePrescription module receives Surescripts message at `/eprescriptions/rx` endpoint and saves it to Aidbox.
+Resources created:
+- `Provenance` - solely as a record that an event has occurred.
```

Comment:
We should decide what type of syntax to use for resources. Should it be backticks or bold (double asterisks)? I used whatever was there in NewRx docs (double asterisk), so it might be necessary to change things there too if we go down the backticks road.

Reply:
I didn't pay much attention to it. Maybe bold is better, I'm gonna fix that